src/fides/ctl/core/config/security_settings.py

```diff
@@ -24,16 +23,16 @@ class SecuritySettings(FidesSettings):
     rate_limit_prefix: str = "fides-"
     aes_encryption_key_length: int = 16
     aes_gcm_nonce_length: int = 12
-    app_encryption_key: str
+    app_encryption_key: str = ""
```

Comment:
With this being set to an empty string won't we need a check any time it is used to make sure it is set to something other than `""`? This seems the same as setting it to a default string which is what we wanted to remove, the difference being now we could check for `""` with this.

Reply:
if all of the values get checked for existence before the webserver exists, technically no right?

We're moving where we're doing the checking from the config itself to that function that gets run before the webserver

I agree this feels janky, but I think it is reasonable as a fix for now

Reply:
we're using boolean checks (`if value:` == 'if "":`), which will come up as `False` when doing the checks (in theory?)